Changes for issue #115 and a few I spotted on the way

PeterParslow opened the pull request:
…asciidoc

'data resource' generally to 'resource'

PeterParslow committed:
Update 1052-metadata-guidelines-for-geospatial-data-resources-part-1.asciidoc

'data resource' generally to 'resource'

PeterParslow commented:
1040 is very general; I've left it unchanged.
1047 the phrase is part of the title of a reference document, so should not change
1047 & 1049 the phrase is part of the title of the page, which itself was inherited from a pre-existing document. I've left these pending some wider discussion about restructuring / renaming those "old" part 2 & part 3 ideas.
1051 is basically an old (GEMINI v2.2) "home page" - again, I've left that as is.

archaeogeek reviewed (commented):
Do we need to update the date in Line 4 too?

PeterParslow commented:
> Do we need to update the date in Line 4 too?

Interesting question: should we reserve that for changes which actually change the meaning, or should we update it even for changes like this (same meaning, different words - if you get my drift...)?

PeterParslow committed:
Update 1053-common-metadata-errors-uk-location-discovery-metadata-service.asciidoc

Fix for issue #115 and also some updates to section 1

PeterParslow committed:
Update 1054-operational-guide for issue #115

PeterParslow committed:
Update 1053-common-metadata-errors-uk-location-discovery-metadata-service.asciidoc - last update date

PeterParslow committed:
Update 1056-glossary for issue 115 and a few other things

such as changes of web site; UK Location Programme no longer exists

PeterParslow committed:
Update 1250-element-summary Topic category

"data resource" to "resource"

PeterParslow committed:
Update abstract.asciidoc - "data resource" -> "resource"

PeterParslow committed:
Update additionalinformation.asciidoc "data resource" -> resource

PeterParslow committed:
Update alternatetitle.asciidoc "data resource" -> resource

PeterParslow committed:
Update boundingbox.asciidoc "data resource" -> resource

PeterParslow committed:
Update coupledresource.asciidoc "data resource" -> resource

PeterParslow committed:
Update dataformat.asciidoc "data resource" -> resource

PeterParslow committed:
Update datasetlanguage.asciidoc "data resource" -> resource

PeterParslow committed:
Update datasetreferencedate.asciidoc "data resource" -> resource

PeterParslow committed:
Update extent.asciidoc "data resource" -> "resource"

And removed three of the four examples because they no longer worked (data.os.uk, ONS digital, local government register)

PeterParslow committed:
Update keyword.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update limitationsonpublicaccess.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update lineage.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update qualityscope.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update resourceidentifier.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update responsibleorganisation.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update spatialreferencesystem.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update spatialresolution.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update temporalextent.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update title.asciidoc "data resource" -> "resource"

PeterParslow committed:
Update topiccategory.asciidoc "data resource" to "resource"

and made change history "collapsible" (hopefully)

PeterParslow committed:
Update useconstraints.asciidoc "data resource" to "resource"

PeterParslow committed:
Update verticalextent.asciidoc "data resource" to "resource"

PeterParslow committed:
Update Readme.md : removed outdated "notes on publishing"

PeterParslow committed:
Update Readme.md - remove 2021 migration status column

PeterParslow committed:
Update Readme.md

PeterParslow committed:
Update Readme.md
